romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -91,6 +92,12 @@ def create_image_model(input_model, image_info):
             }
     out_node = rds.WfiImage(inst)
     im = rdd.ImageModel(out_node)
+
+    # trim off border reference pixels from science data, dq, err
+    im.data = im.data[:, 4:-4, 4:-4]
+    im.dq = im.dq[:, 4:-4, 4:-4]
+    im.err = im.err[:, 4:-4, 4:-4]
```

Comment:
Don't we also need to trim var_poisson, var_rnoise?

Reply:
Gack, never mind, these will change when the reference files are updated.

Reply:
`im` here is the flattened 2D image.